core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -349,7 +348,11 @@ private Shorthand() {}
     public static BinaryValueExpression mapRight(final BiFunction<ValueExpression, ValueExpression, BinaryValueExpression> func, final SingleValueExpression leftExpand, final ValueExpression right) { return func.apply(exp(leftExpand, count(right)), right); }
 
     /** @see Bytes */ public static ValueExpression bytes(final ValueExpression operand) { return new Bytes(operand); }
-    /** @see Scope */ public static ValueExpression scope(final ValueExpression scopedValueExpression, final SingleValueExpression scopeSize) { return new Scope(scopedValueExpression, scopeSize); }
+
+    /** @see Ref */ public static NameRef scope(final NameRef operand) { return scope(operand, con(0)); }
```

Comment:
Do we think these shorthands (with the default scope of 0) could be useful?

Reply:
Maybe, but I'm not sure about the name, maybe `scopedRef`? But maybe all `Ref`s that have the `scopeSize`-argument should be called that? I haven't given it much thought yet, but it might be worth considering.

Reply:
What would be a typical use case for a `Ref` with scope of 0?

Reply:
That means the most local scope that is possible, i.e. only values in the current seq.
I think it is the scope level that will be used most, hence the shorthand for it.

Reply:
> Maybe, but I'm not sure about the name, maybe `scopedRef`? But maybe all `Ref`s that have the `scopeSize`-argument should be called that? I haven't given it much thought yet, but it might be worth considering.

If we would call it `scopedRef`, then usage would look like `scopedRef(ref("name"))`. I like to keep it `scope` shorthand, especially since it is now always a `ref`, so including `Ref` is redundant to me.